tests/test_algorithm/test_configuration/test_maps/run_map_test.py

```diff
@@ -0,0 +1,8 @@
+import sys
+sys.path.append('../../../../src')
+
+import test_dense_map as tst
+
+t = tst.TestDenseMap()
+
+t.test_convert_to_sparse_map()
```

Comment:
No newline

Reply:
Line 5 is needed since its after an import, no?

Reply:
Not line 5, I meant at the end of the file

Reply:
Oh cool, just changed this.